packages/client/src/hooks/useForm.tsx

```diff
@@ -0,0 +1,99 @@
+import {ChangeEvent, useState} from 'react';
+import {Fn} from 'client/src/types';
+import {REG_EMAIL, REG_LOGIN, REG_PASSWORD} from 'client/src/regExp';
+import {emailError, loginError, passwordError} from 'client/src/errors/errors';
+
+export enum InputsNames {
+	login = 'login',
+	email = 'email',
+	password = 'password'
+}
+
+function omit(obj: Record<string, string>, fields: string) {
+	const result = {};
+	Object.keys(obj).forEach(element => {
+
+		if(!fields.includes(element)) {
+			result[element] = obj[element];
+		}
+	});
+	return result;
+}
```

Comment:
Такого рода утилитарные функции неплохо бы вынести в отдельную папку `utils` чтобы можно было переиспользовать. И хорошей практикой было бы сделать типизацию немного более конкретной через джененрики, чтобы на выходе получать правильный тип, например:
```suggestion
function omit<T extends Record<string, any>, T2 extends [...(keyof T)[]]>(
  obj: T,
  fields: T2
): {
  [K2 in Exclude<keyof T, T2[number]>]: T[K2];
} {
  const result = {} as {
    [K2 in Exclude<keyof T, T2[number]>]: T[K2];
  };
  Object.keys(obj).forEach((key) => {
    if (!fields.includes(key)) {
      result[key] = obj[key];
    }
  });
  return result;
}
```